lumigator/python/mzai/jobs/evaluator_lite/schemas.py

```diff
@@ -0,0 +1,67 @@
+"""Schema file that is loaded by the backend to validate the input/output data.
+This file must only depend on pydantic and not on any other external library.
+This is because the backend and this job will be running in different environments.
+"""
+
+from pydantic import BaseModel, ConfigDict, Field
+
+
+class DatasetConfig(BaseModel):
+    path: str
+    model_config = ConfigDict(extra="forbid")
+
+
+class ModelConfig(BaseModel):
+    path: str
+    model_config = ConfigDict(extra="forbid")
+
+
+class EvaluationConfig(BaseModel):
+    metrics: list[str] = Field(default=["rouge", "meteor", "bertscore"])
+    max_samples: int = 0
+    return_input_data: bool = False
+    return_predictions: bool = False
+    storage_path: str
+    model_config = ConfigDict(extra="forbid")
+
+
+class EvalJobConfig(BaseModel):
+    name: str
+    dataset: DatasetConfig
+    model: ModelConfig
```

Comment:
historically Pydantic has disallowed using `model` as a field name due to namespace conflicts with Model objects and might complain here

Reply:
Hmm. It's not throwing any complaints. I know it doesn't like anything with model + an underscore like `model_foo` so maybe it doesn't care as long as the underscore isn't there 🤷